### PR TITLE
Ddcnls 6575

### PR DESCRIPTION
**Pull request by matthew-mcnally**

### CODE REVIEW

File: app/repositories/BulkCalculationRepository.scala [hunk 1]
```diff
@@ -452,7 +452,7 @@ class BulkCalculationMongoRepository @Inject()(metrics: ApplicationMetrics)(impl
           }
 
           val insertResult = proxyCollection.insert(strippedBulk).flatMap {_ =>
-            proxyCollection.insert[ProcessReadyCalculationRequest](ordered = false).many(bulkDocs)
```
Comment (daniel-manning) [inline app/repositories/BulkCalculationRepository.scala]: why remove the type annotation?

Comment (matthew-mcnally) [reply app/repositories/BulkCalculationRepository.scala]: Type put back in now

Comment (daniel-manning) [reply app/repositories/BulkCalculationRepository.scala]: Not what I meant. The type on the insert (as it was before)

Comment (matthew-mcnally) [reply app/repositories/BulkCalculationRepository.scala]: done now

**Pull request merged**